database/ioc/src/Makefile

```diff
@@ -1,4 +1,4 @@
-TOP=../..
+TOP=../../..
```

Comment:
On 08/16/2016 03:52 PM, Andrew Johnson wrote:
>
> In database/ioc/src/Makefile <https://github.com/epics-base/pvExampleTestCPP/pull/1#discussion_r75035740>:
>
> > @@ -1,4 +1,4 @@
> > -TOP=../..
> > +TOP=../../..
>
> There should be an entry in the |database/configure/RELEASE| file pointing to |$(TOP)/..|, which is how this is 
> handled in the other embedded tops in this module.
>
I've submitted an updated patch w/ that strategy.
>
> Given that this module has been superceded by moving its code into the enhanced exampleCPP module we don't really want 
> to spend much time on this, and I've suggested that Marty remove everything except an explanatory README.md file from 
> the master branch to document that. I'd be happy to merge a modified version with your other changes into the 
> release/4.1 branch for users still on the V4.5.0.x series if you think that's worth doing.
>
It would help to have this module and any others that are being superceded
to be cleared out so we don't spend time trying to build them.
In the meantime I think it would be worthwhile to at least ensure these older
examples build successfully.

I got into this as neither the 4.5.0.1 nor 4.5.0.2 v4 downloads
would build for me.   The problems were all static link errors as I
recall, due to LIB directives either missing or out of order.
I got the same errors w/ RHEL5 gcc4.1.2 and RHEL6 gcc4.4.7, so
maybe the development was done w/ more current gcc versions.

Instead of downloading 4.6.0-pre1, since it didn't appear to be stable
yet, I just cloned the git repos and started building them.
Perhaps it's time to release 4.6.0.1?

Cheers,
- Bruce